app/src/main/res/drawable/ic_telegram.xml

```diff
@@ -0,0 +1,22 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="28dp"
```

Comment:
why 28dp? isn't the default 24dp (or something else that can be devided by 8) ... I would strongly prefer if we stick to the grid of 8 (or at least 4)

Reply:
those are the values from our mocks. I will use 24dp